### PR TITLE
Fix invalid restrictNSMDeps check, add dependent repositories

### PR DESCRIPTION
**Pull request by Bolodya1997**

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -138,7 +138,17 @@ jobs:
...
+        repository:
+          - sdk-vppagent
+          - sdk-kernel
+          - sdk-sriov
+          - cmd-nsmgr
+          - cmd-nsmgr-proxy
+          - cmd-registry-memory
+          - cmd-registry-proxy-dns
+          - cmd-forwarder-vppagent
+          - cmd-forwarder-sriov
```
Comment (denis-tingajkin) [inline .github/workflows/ci.yaml:150]: Note: sdk-sriov should update md-forwarder-sriov

**Review by denis-tingajkin — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (Bolodya1997) [reply .github/workflows/ci.yaml]: Fixed this and some more

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -138,7 +138,13 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        repository: [cmd-registry-memory, cmd-registry-proxy-dns, sdk-kernel, sdk-vppagent, cmd-nsmgr]
+        repository:
+          - sdk-vppagent
```
Comment (denis-tingajkin) [inline .github/workflows/ci.yaml:142]: I think we can also add `cmd-nse-icmp-responder`

Comment (Bolodya1997) [reply .github/workflows/ci.yaml:142]: Yep, done.

**Pull request merged**